BoxContentSDK/BoxContentSDK/BOXURLSessionManager.h

```diff
@@ -250,6 +250,19 @@ totalBytesExpectedToSend:(int64_t)totalBytesExpectedToSend;
  */
 - (BOOL)cleanUpBackgroundSessionTaskIfExistForUserId:(NSString *)userId associateId:(NSString *)associateId error:(NSError **)error;
 
+/**
+ * Clean up background session from cache and invalidate it if possible,
+ * i.e. if the background session to be cleaned up is not the current
+ * background session and has no pending tasks
+ *
+ * @param backgroundSessionID   background session ID to clean up
+ * @param error error cleaning up this background session
+ *
+ * @return YES if successfully clean up, NO otherwise
+*/
+- (BOOL)cleanUpBackgroundSessionIfPossibleGivenID:(NSString *)backgroundSessionID
```

Comment:
Nit: consider renaming this to simply `cleanUpBackgroundSession:`
Given that the result is a BOOL, for success/failure, the "if possible" is somewhat redundant.